### PR TITLE
Add nodepool flag for linbit

### PR DESCRIPTION
**Pull request by dramaturg**
This adds handling addon flags indentically to what we do for clusters.

### CODE REVIEW

**Comment by exo-cedric** [comment]
May I suggest we'd rather introduce a --partitioning (or --disk-part or ...) parameter, allowing to cover as many partitioning use cases as we might want (default, linstore, luks, etc.)

File: cmd/sks_nodepool_add.go [hunk 1]
```diff
@@ -63,6 +68,24 @@ func (c *sksNodepoolAddCmd) cmdRun(_ *cobra.Command, _ []string) error {
...
+
+		if len(addOns) > 0 {
+			list := make([]string, 0)
+			for k := range addOns {
+				list = append(list, k)
+			}
+			v = &list
+		}
+		return
+	}()
```
Comment (7fELF) [inline cmd/sks_nodepool_add.go:87]:
```suggestion
	addOns := map[string]bool{
		"linbit": c.Linbit,
	}

	nodepool.AddOns = &[]string{}
	for k, v := range addOns {
		if v {
			*nodepool.AddOns = append(*nodepool.AddOns, k)
		}
	}
```

(tested)

File: cmd/sks_nodepool_add.go [hunk 2]
```diff
@@ -9,6 +9,10 @@ import (
 	"github.com/spf13/cobra"
 )
 
+var (
+	sksNodepoolAddonLinbit = "linbit"
+)
+
```
Comment (7fELF) [inline cmd/sks_nodepool_add.go:15]:
```suggestion
```

**Review by 7fELF — APPROVED** [review]

**Comment by dramaturg** [comment]
> May I suggest we'd rather introduce a --partitioning (or --disk-part or ...) parameter, allowing to cover as many partitioning use cases as we might want (default, linstore, luks, etc.)

We can still do that if we have multiple options at some point in time.

**Review by PhilippeChepy — APPROVED** [review]

**Pull request merged**